libcontainer/configs/memorypolicy.go

```diff
@@ -0,0 +1,29 @@
+package configs
+
+import "golang.org/x/sys/unix"
+
+//nolint:revive,staticcheck,nolintlint // ignore ALL_CAPS errors in consts from numaif.h, will match unix.* in the future
+const (
+	MPOL_DEFAULT = iota
+	MPOL_PREFERRED
+	MPOL_BIND
+	MPOL_INTERLEAVE
+	MPOL_LOCAL
+	MPOL_PREFERRED_MANY
+	MPOL_WEIGHTED_INTERLEAVE
+
+	MPOL_F_STATIC_NODES   = 1 << 15
+	MPOL_F_RELATIVE_NODES = 1 << 14
+	MPOL_F_NUMA_BALANCING = 1 << 13
+)
+
+// LinuxMemoryPolicy contains memory policy configuration.
+type LinuxMemoryPolicy struct {
+	// Mode specifies memory policy mode without mode flags. See
+	// set_mempolicy() documentation for details.
+	Mode uint
+	// Flags contains mode flags.
+	Flags []uint
```

Comment:
I think, since mode and flags are designed to be ORed, we can simplify things and use a single uint for `mode | flags`.

Reply:
Or at least just use a single `uint` for the flags. The spec uses an array so we can have stringified flags, but here we should just use the OR'd version.

Reply:
Based on these comments, I ended up OR'ing the flags but still keeping a separate field to carry them. This could still help debugging, even if separating the flags from each other is not that easy. (OTOH, currently available flags are such that most likely users don't need to combine them, anyway).